exla/mix.exs

```diff
@@ -59,8 +59,8 @@ defmodule EXLA.MixProject do
 
   defp deps do
     [
-      {:nx, "~> 0.4.2"},
-      # {:nx, path: "../nx"},
+      # {:nx, "~> 0.4.2"},
+      {:nx, path: "../nx"},
```

Comment:
I had to make this change to test. How do you compare and test EXLA with Nx main?

Reply:
By doing exactly this. :)